setup.py

```diff
@@ -181,7 +180,7 @@ def read_version_from_git():
     },
     install_requires=install_requires,
     tests_require=test_requirements,
-    python_requires='>=3.5',
+    python_requires='>=3.6',
```

Comment:
What are the breaking changes that make it impossible to run this project with the version 3.5 of the interpreter?

Reply:
It's the `encoding` argument in the `Popen`call:
> New in version 3.6: encoding and errors were added.